src/library/scala/util/Properties.scala

```diff
@@ -13,6 +13,28 @@ package util
 import java.io.{ IOException, PrintWriter }
 import java.util.jar.Attributes.{ Name => AttributeName }
 
+sealed abstract class JavaSpecVersion extends Ordered[JavaSpecVersion] {
```

Comment:
If you put it that way, the only thing anyone ever does is compare bits. :8ball: